pocs/tests/test_dispatch_scheduler.py

```diff
@@ -82,6 +98,45 @@ def test_get_observation(scheduler):
     assert isinstance(best[1], float)
 
 
+@pytest.fixture
+def temp_file():
+    temp_file = 'temp'
+    with open(temp_file, 'w') as f:
+        f.write('')
+
+    yield temp_file
+    os.unlink(temp_file)
+
+
+def test_get_observation_reread(field_list, observer, temp_file):
+    time = Time('2016-08-13 10:00:00')
+
+    # Write out the field list
+    temp_field_file = temp_file
+    with open(temp_field_file, 'w') as f:
+        f.write(yaml.dump(field_list))
+
+    scheduler = Scheduler(observer, fields_file=temp_field_file, constraints=constraints)
+
+    # Get observation as above
+    best = scheduler.get_observation(time=time)
+    assert best[0] == 'HD 189733'
```

Comment:
FYI, I'm a bit disturbed by the assumptions in a number of these tests about time, location & fields. I think I've said before that I'd like separate, simple yaml files for tests, and preferably different such yaml files for different tests (or perhaps fragments combined together to reduce the number of such files needed).

Reply:
Agreed that it needs to be thought out and made a little more robust.